src/main/java/net/glowstone/GlowServer.java

```diff
@@ -476,6 +477,13 @@ public void start() {
         nameBans.load();
         ipBans.load();
 
+        try {
+            LootingManager.load();
+        } catch (Exception e) {
```

Comment:
Is it possible to provide a more useful error message? Maybe be more fine with the exception catching, or include the exception in the logger message.